ui/components/ConnectionWizard/helpers/jsonParser.js

```diff
@@ -0,0 +1,3 @@
+export const parseJson = (json) => {
+  return JSON.parse(json);
```

Comment:
@EraKin575 Not exactly this, but instead adding error handling here itself, so that if the input is undefined or null the error handling is done inside this function itself.

Reply:
```suggestion
try {
  return JSON.parse(json);
  } catch (e) {
  console.error(e);
    return {};
  }
```